src/clr-angular/forms/styles/_select.clarity.scss

```diff
@@ -97,8 +97,12 @@
     }
   }
 
-  .clr-error select[multiple] {
+  .clr-error select {
     border-color: $clr-forms-invalid-color;
+
+    &:not([multiple]) {
```

Comment:
You don't really need this part, as it automatically gets it from the `select` definition on line 11 (include itself is on line 14).

What I observe is that while focused the multiple select looks a bit unexpected:
<img width="156" alt="Screen Shot 2019-07-19 at 1 13 10 PM" src="https://user-images.githubusercontent.com/1798828/61528272-007ab900-aa27-11e9-9b18-5812cd28e62f.png">

See the bottom blue line (with animation). I guess this is what you tried to avoid with the `:not([multiple])` but actually, previously before your update, we did not have any focus indication for multiple selects in error. Now we have. 

I'd like to see some comment from the team and UX before we proceed.

Reply:
Hi Ivan,

Thanks for the feedback. 

Earlier, we have the error/red border only for multiple select, not for the single select box.
As per the clarity standard, we are following the red border and focus animation on error elements. I try to add both color and animation to all select elements. But the animation doesn't look good for multiple select elements.

That is the reason, I added the red border and focus animation for single select, whereas, multi-select kept as-is with the only border.

Regarding .angulardoc.json, the file got checked-in accidentally along with these files. I didn't notice it. 
I will make sure to not happen in the future.